flybywiresim/a32nx
# feat: exterior engine sound rework and other changes

## Summary of Changes


I have completely reworked the exterior engine startup, spool up, and flyby sounds.

-The startup now has a better ignition sample, and an overall higher quality sound
-The spool up has many more layers and now is very realistic
-Flyby has a better exhaust sound and an overall more realistic sound
-Reverse sound is much better on exterior

Interior has got some small changes too.

-Startup has a completely reworked "before ignition" sound
-It now has a better transition between the ignition cutout and the idle sound (more easily heard in the cabin)
-It has a slight whale call sound between 20% and 27% n1
-It has a higher quality buzzsaw sound
-Reverse is a bit louder in cockpit now
-New nosewheel rotation/liftoff sound

Now for other changes:

-I have changed the APU initial startup sound to have a bit faster igniters, as a reference from komp. Before they were way too slow
-Pack sounds now come from the pack outflow valves
-Gear sounds now differ slightly for gear up and down
-The taxi sound now is a little higher quality, i removed some of the lower quality clicks

## References


Used a bunch of youtube videos and references from engineers. Apu video from komp and engine references from a lot of youtube videos.


## Additional context


Let me know of any suggestions or tweaks I can make, I will try to make them as quick as I can!


Discord username (if different from GitHub):

Boris#9134

## Testing instructions


1. Start batteries, then press APU master and starter switch and immediately go to exterior view and listen for igniters
2. Start APU bleed and listen for the pack sounds and that they are coming from the outflow valves
3. Start engine, preferrably listen from the cabin, and listen to the entire sequence. Then shut down the engine and listen again from the exterior view
4. Start both engines, and in the cabin listen for the quiet whale call sound at just above idle thrust
5. Go to exterior and spool up the engines to TOGA slowly and listen to the spool up
6. Take off and listen to gear up sound and see that it is different from the gear down
7. During takeoff, position your drone camera farther from the aircraft and turn off Follow Mode so the aircraft flies by and you hear the flyby sounds, try doing a flyby at 80-90% n1, and at around 50% n1 to hear how it sounds.
8. Listen to the buzzsaw at takeoff
9. On approach, check gear down sound
10. On landing, listen to reverse thrust from interior and exterior to confirm that the sound was louder on the interior and that it is much better on exterior than before


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
